packages/fxa-payments-server/src/components/Home/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Elements } from 'react-stripe-elements';
+import { selectors, actions } from '../../store';
+import { mapToObject } from '../../utils';
+
+import SubscriptionForm from '../SubscriptionForm';
+
+export const Home = ({
+  config = {},
+  accessToken = '',
+  isLoading = true,
+  lastError = null,
+  profile = {},
+  plans = [],
+  subscriptions = [],
+  productId = null,
+  plansByProductId = [],
+  createSubscription,
+  cancelSubscription,
+}) => {
+  if (!accessToken) {
+    return <h1>Need an #accessToken=... parameter!</h1>
+  }
+  return (
+    <div>
+      <a href={`${config.CONTENT_SERVER_ROOT}/settings`}>&#x2039; Back to FxA Settings</a>
+
+      {profile.loading && <h1>(profile loading...)</h1>}
```

Comment:
Yeah, at one point I had a full page "loading" message if any of the fetch-based state was loading, rather than doing it granularly like this. We can probably just do that.